src/app/shared/components/city-search/city-search.component.html

```diff
@@ -0,0 +1,9 @@
+<form class="fillContent flexType" ng-submit="$ctrl.routingFunctions.search($ctrl.city, $ctrl.latestSearches)" novalidate>
```

Comment:
on the view/template is preferred to call controller functions, not use services methods directly, is not intuitive
just wrap this statement on a controller method